lib/get-build-template-compiler-tree.js

```diff
@@ -38,7 +37,7 @@ module.exports = function buildTemplateCompilerTree(packages) {
   var compiled = concatenateES6Modules(trees, {
     includeLoader: true,
     bootstrapModule: 'ember-template-compiler',
-    vendorTrees: mergeTrees(vendorTrees),
+    vendorTrees: mergeTrees(vendorTrees, {overwrite: true}),
```

Comment:
dependencies step on each other if we don't use overwrite, I'm unsure why exactly.